<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:#F9F7F7;
}


#copyRight_widget{
background-color:#5C7CFA;
padding-right:4px;
padding-left:4px;
/*
border-bottom-left-radius:10px;
border-bottom-right-radius:10px;
border-top-left-radius:0px;
border-top-right-radius:0px;*/

}
#copyRight_widget_label_2{
background-color:#5C7CFA;
color:white;
padding-right:4px;
/*
border-bottom-right-radius:10px;
border-top-right-radius:none;
*/
}
#copyRight_widget_label_1{
background-color:#5C7CFA;
color:white;
padding-right:4px;
/*
border-bottom-left-radius:10px;
border-top-left-radius:0px;
*/
}


QGroupBox{
border-color: #5C7CFA;
border-style:solid;
border-width:1px;
margin-top:6px;
}
QGroupBox:title{
subcontrol-origin:margin;
left:10px;
bottom:5px;
padding: 0 5px;
}
QLineEdit{
border:none;
}


/*btn*/

#annuler_btn {
background-color:#5C7CFA;
color:white;
border-radius: 5px;
border:none;
padding-right:8px;
padding-left:8px;
padding-top:2px;
padding-bottom:2px;
}
#connect_btn {
background-color:#5C7CFA;
color:white;
border-radius: 5px;
border:none;
padding-right:8px;
padding-left:8px;
padding-top:2px;
padding-bottom:2px;
}

#connect_btn:hover{
background-color:#7993F8;
color:#fbfafa;
}
#annuler_btn:hover{
background-color:#7993F8;
color:#fbfafa;
}

#connect_btn:checked{
background-color:#3F72AF;
color:#112D4E;
}
#annuler_btn:checked{
background-color:#3F72AF;
color:#112D4E;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: rgb(255, 255, 255);
border-radius:5px;
padding:4px;
}

QPushButton{
border:none;
}
QLabel{
border:none;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="leftMargin">
       <number>40</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>40</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/assets/icons/output-onlinepngtools (1).png</normaloff>:/assets/icons/output-onlinepngtools (1).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>103</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/assets/icons/output-onlinepngtools (2).png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="copyRight_widget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="copyRight_widget_label_1">
          <property name="font">
           <font>
            <family>Segoe UI Light</family>
            <pointsize>6</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Azer Hasnaoui copyRightÂ©</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="copyRight_widget_label_2">
          <property name="font">
           <font>
            <family>Segoe UI Light</family>
            <pointsize>7</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>2024</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="mainLoginWidget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Bienvenue</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_1">
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Login</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="login_lineEdit">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="placeholderText">
            <string>Nom de l'utilisateur</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>mot de passe</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="pass_lineEdit">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>Mot de passe</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="msgLogin">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: red;</string>
        </property>
        <property name="text">
         <string>mot de passe ou nom d'utilisateur incorrect !</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>51</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="btn_widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>40</number>
         </property>
         <property name="rightMargin">
          <number>40</number>
         </property>
         <item>
          <widget class="QPushButton" name="annuler_btn">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color: rgb(92, 124, 250);
	color: rgb(255, 255, 255);
}

QPushButton:hover{
	color: rgb(92, 124, 250);
	background-color: rgb(255, 255, 255);
}

</string>
           </property>
           <property name="text">
            <string>Annuler    </string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/assets/icons/cancel-30.png</normaloff>
             <normalon>:/assets/icons/cancel-30+active.png</normalon>:/assets/icons/cancel-30.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="connect_btn">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color: rgb(92, 124, 250);
	color: rgb(255, 255, 255);
}

QPushButton:hover{
	color: rgb(92, 124, 250);
	background-color: rgb(255, 255, 255);
}

</string>
           </property>
           <property name="text">
            <string>Connecter    </string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/assets/icons/connect-30.png</normaloff>
             <normalon>:/assets/icons/connect-30_active.png</normalon>:/assets/icons/connect-30.png</iconset>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>annuler_btn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
